ci/kind/test-e2e-kind.sh

```diff
@@ -168,7 +168,7 @@ fi
 COMMON_IMAGES_LIST=("registry.k8s.io/e2e-test-images/agnhost:2.29" \
                     "projects.registry.vmware.com/antrea/busybox"  \
                     "projects.registry.vmware.com/antrea/nginx:1.21.6-alpine" \
-                    "projects.registry.vmware.com/antrea/perftool")
+                    "projects.registry.vmware.com/antrea/toolbox:1.1-0")
```

Comment:
Question: why is a suffix `-0` needed here?

Reply:
This is explained here: https://github.com/antrea-io/antrea/issues/5169#issuecomment-1603420154

This is a revision number which is automatically updated by the image building job when the image is rebuilt (e.g., because the base ubuntu image has been updated). The full tag (`<VERSION>-<REVISION>`) is immutable, which is good when using the Harbor registry, as the CDN doesn't handle mutable tags correctly. There are other (mutable) tags available (`latest`, `1.1`) for non-Harbor users.
I didn't go with `1.1.0` because I wanted to emphasize the auto-generated aspect of the revision.